### PR TITLE
Test on PHP 7.4

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

**Comment by ruflin** [comment]
@deguif Could you take a look at the test failure? It might tell us actually about a potential bug?

File: test/Elastica/ClientTest.php [hunk 1]
```diff
@@ -745,7 +745,7 @@ public function testUpdateWithInvalidType()
         try {
             $client->updateDocument(1, $badDocument, $index->getName());
             $this->fail('Tried to update using an object that is not a Document or a Script but no exception was thrown');
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```
Comment (deguif) [inline test/Elastica/ClientTest.php]: @ruflin That should fix the test. As a PHP error was thrown ;)

**Comment by ruflin** [comment]
Great to have it now also tested on 7.4

**Comment by deguif** [comment]
Yes, and we will have to update the docker tag, once GA release is out.

**Pull request merged**